lib/ui/compositing.dart

```diff
@@ -272,10 +272,10 @@ class SceneHost extends NativeFieldWrapperClass2 {
   /// to the child.
   ///
   /// The scene host takes ownership of the provided export token handle.
-  SceneHost(int export_token_handle) {
+  SceneHost(dynamic export_token_handle) {
```

Comment:
Yeah, I want to add some documentation, and even some tests at some point soon.

I don't actually understand this API well. It's to do with Mozart's integration with Flutter. Perhaps @j9brown could explain it better than me.